[2025-01-05T21:44:41.582+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-01-05T21:44:41.598+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_banking_tinkoff.load_db manual__2025-01-05T21:44:40.549683+00:00 [queued]>
[2025-01-05T21:44:41.603+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_banking_tinkoff.load_db manual__2025-01-05T21:44:40.549683+00:00 [queued]>
[2025-01-05T21:44:41.603+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-01-05T21:44:41.610+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_db> on 2025-01-05 21:44:40.549683+00:00
[2025-01-05T21:44:41.615+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'raw_banking_tinkoff', 'load_db', 'manual__2025-01-05T21:44:40.549683+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/raw_banking_tinkoff.py', '--cfg-path', '/tmp/tmp4stm_yzo']
[2025-01-05T21:44:41.616+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=751) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-05T21:44:41.617+0000] {standard_task_runner.py:91} INFO - Job 94: Subtask load_db
[2025-01-05T21:44:41.617+0000] {standard_task_runner.py:63} INFO - Started process 756 to run task
[2025-01-05T21:44:41.643+0000] {task_command.py:426} INFO - Running <TaskInstance: raw_banking_tinkoff.load_db manual__2025-01-05T21:44:40.549683+00:00 [running]> on host 6d9883a799ac
[2025-01-05T21:44:41.685+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='nmashtakov' AIRFLOW_CTX_DAG_ID='raw_banking_tinkoff' AIRFLOW_CTX_TASK_ID='load_db' AIRFLOW_CTX_EXECUTION_DATE='2025-01-05T21:44:40.549683+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-05T21:44:40.549683+00:00'
[2025-01-05T21:44:41.686+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-01-05T21:44:41.829+0000] {logging_mixin.py:188} INFO - Нет данных для загрузки
[2025-01-05T21:44:41.829+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-01-05T21:44:41.829+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-01-05T21:44:41.834+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=raw_banking_tinkoff, task_id=load_db, run_id=manual__2025-01-05T21:44:40.549683+00:00, execution_date=20250105T214440, start_date=20250105T214441, end_date=20250105T214441
[2025-01-05T21:44:41.878+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-01-05T21:44:41.889+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
